### DISCUSSION TITLE
Enlarge DatePicker Calendar

### DISCUSSION THREAD

**Discussion by adrien-jaquiery** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3081] [Q&A]
Hi,
I am trying to enlarge the calendar of the datepicker for touchscreen use but couldn't achieve it so far.
I managed to enlarge it with the default WPF style but not with the one you provide. How can I achieve that?

**Comment by nicolaihenriksen** [comment]
@adrien-jaquiery Can you elaborate a bit on how you want it enlarged? Perhaps some screenshots of the default WPF style that you have enlarged?

The `Calendar` control is a somewhat complex beast where there are styles for `CalendarButton` and `CalendarDayButton` inside which have a lot of effect on the overall sizing of the control; these do after all occupy most of the real estate of the control.

**Comment by MichelMichels** [comment]
Closed due to no feedback from OP
